src/Redux/Store/Randomizer/BasicNemesisCards/RandomCards/types.ts

```diff
@@ -1,13 +1,11 @@
 import { ActionsUnion } from '@martin_hotell/rex-tils'
 
-import * as types from 'types'
-
 import { actions } from './actions'
 
 export type State = Readonly<{
-  Tier1: ReadonlyArray<types.BasicNemesisCard>
-  Tier2: ReadonlyArray<types.BasicNemesisCard>
-  Tier3: ReadonlyArray<types.BasicNemesisCard>
+  Tier1: ReadonlyArray<{ id: Readonly<string> }>
```

Comment:
string can't be mutated anyway, technically the parent object could.

So if you would like to go for more savety here, a type like this would be more useful:

```ts
type Tier = ReadonlyArray<ReadOnly<{ id: string }>>
```